kernel_hardening_checker/__init__.py

```diff
@@ -48,7 +48,7 @@ def detect_arch(fname, archs):
 
 def detect_kernel_version(fname):
     with _open(fname, 'rt', encoding='utf-8') as f:
-        ver_pattern = re.compile("# Linux/.+ Kernel Configuration$")
+        ver_pattern = re.compile("# Linux/.+ Kernel Configuration$|^Linux version .+")
         for line in f.readlines():
```

Comment:
It would be useful to print an advice about the `-v` option in case of failure of the default version parsing.

It is the case that you are fixing.